contributing_to_docs/doc_guidelines.adoc

```diff
@@ -139,6 +140,8 @@ separated, global use of `{product-version}` across all branches is probably
 less useful, but it is available if you come across a need for it. Just consider
 how it will render across any branches that the content appears in.
 
+Markup, such as in commands, persists in headings.
```

Comment:
What sorts of things would necessitate markup in headings? I think we should avoid putting file names or variable names into headings wherever possible.